.github/workflows/self-service-labeler.yml

```diff
@@ -0,0 +1,40 @@
+name: Issue Opened Self-Service Labeler
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  triage:
```

Comment:
I believe this is the name of the job. Could maybe change to something like `label` or `add-labels`?

Reply:
Done.